Library/Library/AddBook.aspx.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Library.Data;
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Web;
+using System.Web.UI;
+using System.Web.UI.WebControls;
+
+
+
+namespace Library
+{
+    public partial class AddBook : System.Web.UI.Page
+    {
+        protected void Page_Load(object sender, EventArgs e)
+        {
+
+        }
+
+        protected void saveBook()
+        {
+            string bookTitle = BookTitle.Text;
+            string ISBN = ISBNGet.Text;
+            int? authorsId = int.Parse(AuthorListChose.SelectedValue);
+            int? libraryId = int.Parse(LibraryList.SelectedValue);
```

Comment:
It's okay to just use the `int` type for these variables as the `int.Parse` method returns an `int`.